docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+
+In the previous example, we could have regions in which the 'socialist' party won. This category wasn't placed in the `buckets` function so it will fall back to the `others` bucket.
+
+The `others` bucket will be colored gray by default. However, it's possible to override this behavior by providing a third parameter to `ramp`: `ramp(buckets($winner, ['conservatives', 'progressives'], [red, blue], white)`.
+
+<div class="example-map">
+    <iframe
+        id="election-others-bucket"
+        src="/developers/carto-vl/examples/maps/guides/ramp/election-others-bucket.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-election---others-bucket">View my source code!</a>
+</div>
+
+##### Showing the most common categories: `top`
+
+If we don't care about which colors get each category, but we don't want to color every category in the dataset, we can use `top` to group all uncommon categories in the *others bucket*.
+
+`top($cause, 5)` function will keep the five most common categories (regarding the entire dataset) and will group the rest into the *others bucket*.
+
+Let's see this with a dataset of US railroad accidents.
+
+<div class="example-map">
+    <iframe
+        id="accidents-top"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-top.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-railroad-accidents---top-expression">View my source code!</a>
+</div>
+
+##### Showing every category without selecting each color
+
+Sometimes, we don't care about the correspondence between colors and categories nor about having too many categories. This is particularly useful for getting quick feedback and exploring a dataset, but it is of reduced utility in later stages.
+
+For this case, we can request to see every category by putting the property as the `ramp` input without enclosing it in a function like `buckets`.
+
+<div class="example-map">
+    <iframe
+        id="accidents-all"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-all.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-railroad-accidents---all-types">View my source code!</a>
+</div>
+
+As you can see, CARTO VL is generating intermediate colors by interpolating the provided colors. This is always done when the provided list of colors doesn't match the number of categories in the input. It's difficult to distinguish colors when there are so many categories, you should try to avoid this form (to use `buckets` or `top`) when this happens.
+
+###### CieLAB interpolation
```

Comment:
Is this a 5th level instead of 6th?

Reply:
By the way, I didn't know we were interpolating this way. It's great!

Reply:
Maybe just a note

Reply:
As you wish!